packages/cra-external-template/template/gitignore

```diff
@@ -0,0 +1,25 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Should this file be called `.gitignore` looks like you are missing a dot

Reply:
No typo here. Ref: https://create-react-app.dev/docs/custom-templates/

> This folder is copied to the user's app directory as Create React App installs. During this process, the file gitignore is renamed to .gitignore.